delivery_auto_refresh/models/sale_order_line.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2024 Jacques-Etienne Baudoux (BCIM) <je@bcim.be>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class SaleOrderLine(models.Model):
+    _inherit = "sale.order.line"
+
+    def _get_protected_fields(self):
```

Comment:
As this is a child model (coming from a o2m), we prefer to not separate it in a different file. Please respect this preference and reduce the diff.